slo-monitor/slo-monitor-pod.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: v1
```

Comment:
One is for running as in-cluster on normal Node, and one is to be used as a manifest file.